### PR TITLE
[Text Fragments article] Add opt-out steps

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add documentation how people can opt out of this feature.

### CODE REVIEW

**Comment by tomayac** [comment]
FYI: @bokand

File: src/site/content/en/blog/text-fragments/index.md [hunk 1]
```diff
@@ -429,6 +429,23 @@ origin trial is running from Chrome version 83 to 85.
 Document-Policy: force-load-at-top
 ```
 
+## Disabling text fragments
+
+In case you really dislike the feature, you can opt out by leveraging the enterprise setting
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:434]:
```suggestion
If you really dislike the feature, you can opt out using the enterprise setting
```

File: src/site/content/en/blog/text-fragments/index.md [hunk 2]
```diff
@@ -429,6 +429,23 @@ origin trial is running from Chrome version 83 to 85.
...
+[`ScrollToTextFragmentEnabled`](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ScrollToTextFragmentEnabled).
+Therefore, on macOS, paste the command below in the terminal.
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:436]:
```suggestion
To use it on macOS, paste the command below in the terminal.
```

File: src/site/content/en/blog/text-fragments/index.md [hunk 3]
```diff
@@ -429,6 +429,23 @@ origin trial is running from Chrome version 83 to 85.
...
+
+```bash
+defaults write com.google.Chrome ScrollToTextFragmentEnabled -bool false
+```
+
+On Windows, follow the documentation on the 
+[Google Chrome Enterprise Help](https://support.google.com/chrome/a/answer/9131254?hl=en).
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:443]:
```suggestion
[Google Chrome Enterprise Help](https://support.google.com/chrome/a/answer/9131254?hl=en) support site.
```

**Comment by tomayac** [comment]
Thanks, Joe. PTAL.

**Pull request merged**